src/util/addBoxPlot.js

```diff
@@ -1,30 +1,20 @@
-export default function addBoxplot
-    (svg
-    ,results
-    ,height
-    ,width
-    ,domain
-    ,boxPlotWidth
-    ,boxColor
-    ,boxInsideColor
-    ,format = '.2f'
-    ,horizontal = true) {
+export default function addBoxplot(chart, group
```

Comment:
webcharts specific now (instead of generic D3 function). Not a problem, just worth noting.